apps/settings/style/icons.css

```diff
@@ -19,6 +19,7 @@ a.menu-item:after {
   right: 0.3rem;
   font-size: 25px;
   line-height: 6rem;
+  opacity: 0.7;
```

Comment:
I agree but we have :active state too that's why I'm not sure color or opacity ...